google/api/annotations.proto

```diff
@@ -113,5 +120,6 @@ message Project {
 // field.
 // (See `resources.proto` in this directory.)
 message Organization {
-  string name = 1 [(google.api.resource).path = "organizations/*"];
+  string name = 1
```

Comment:
This should be a file level resource definition in order to avoid creating a message that is unneeded.

```proto
option (google.api.resource_definition) = {
  name: "Organization",
  path: "organizations/{organization_id}"
}
```